.github/workflows/backend_cd.yml

```diff
@@ -6,12 +6,14 @@ on:
 env:
   env_file: ${{ secrets.env_prod }}
   aws_pem_file: ${{ secrets.aws_pem_file }}
-  docker_password: ${{ secrets.docker_password }}
 jobs:
   Testing:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
         with:
           ref: main 
-      - run: ls && echo "$env_file" > ".env" && echo "$aws_pem_file" > aws.pem && docker login --username swanso15 --password $docker_password && docker build -t swanso15/cupboard_backend:latest . && docker image ls && docker save -o cupboard_backend.tar swanso15/cupboard_backend:latest
+      - name: Create docker image and send to aws
+        run: echo "$env_file" > ".env" && echo "$aws_pem_file" > aws.pem && chmod 600 aws.pem && docker build -t cupboard_backend:latest . && docker save -o cupboard_backend.tar cupboard_backend:latest && scp -i aws.pem -o StrictHostKeyChecking=no cupboard_backend.tar ec2-user@35.183.197.0:/home/ec2-user
```

Comment:
Please break the line for run into multiple lines for readability like how the backend_ci.yml file is right now. (newline wherever && is) i.e.
```
run: |
  echo "$env_file" > ".env"
  echo "$aws_pem_file" > aws.pem
  .
```